api/lib/infrastructure/repositories/challenge-repository.js

```diff
@@ -33,20 +36,36 @@ module.exports = {
       cache.get(cacheKey, (err, cachedValue) => {
         if (err) return reject(err);
         if (cachedValue) return resolve(cachedValue);
-        return _fetchChallenges(cacheKey, resolve, reject, () => true);
+        return _fetchChallenges(cacheKey, resolve, reject);
       });
     });
   },
 
   getFromCompetenceId(competenceId) {
+    const viewNames = {
```

Comment:
N'ayez pas peur.

Reply:
Ah oui, quand même ! :-D
Je m'occupe de faire les modifs pour passer de façon moins statique par une méthode dans le CompetenceRepository.

Reply:
Oki ! Mais fais gaffe, ça existe pas encore. Il faut choper le champ Référence. Le code Python qui m'a permis de générer le dictionnaire :

    data = make_request('/Competences', {'fields': ['Titre', 'Sous-domaine', 'Référence']})
    for record in data['records']:
        domains[record['id']] = record['fields']['Référence']